lib/Common/Memory/Recycler.h

```diff
@@ -1653,6 +1658,14 @@ class Recycler
     {
         return ((collectionState & Collection_ConcurrentSweep) == Collection_ConcurrentSweep);
     }
+
+#if ENABLE_ALLOCATIONS_DURING_CONCURRENT_SWEEP
+    bool AllocationsDuringconcurrentSweepTurnedOn()
```

Comment:
Nit: camel case name here- and I think AllocAllocationDuringConcurrentSweep is a tad bit more compact